src/fr/adrienbrault/idea/symfony2plugin/Settings.java

```diff
@@ -26,12 +26,14 @@
     public static String[] DEFAULT_CONTAINER_PATHS = new String[] {
         "app/cache/dev/appDevDebugProjectContainer.xml",
         "var/cache/dev/appDevDebugProjectContainer.xml",
+        "var/cache/dev/srcDevDebugProjectContainer.xml",
     };
 
     // Symfony 2 and 3 paths
     public static String[] DEFAULT_ROUTES = new String[] {
         "app/cache/dev/appDevUrlGenerator.php",
         "var/cache/dev/appDevUrlGenerator.php",
+        "var/cache/dev/srcDevDebugProjectContainerUrlGenerator.php",   

```

Comment:
are you sure about `DebugProjectContainer` here ? This looks weird to me

Reply:
ooh, yees, seems to typo, will check later for real name

Reply:
well, probably `srcDevUrlGenerator.php`, as the only change is the kernel name (because of the kernel move from `app` to `src`)

Reply:
btw, the comment before should say `Symfony 2, 3 and 4 paths` 😄

Reply:
Looks like path is right.

![default](https://user-images.githubusercontent.com/191082/29967296-a9283436-8f1e-11e7-9f9a-f97dbdd865d5.png)

Reply:
@stof even more, Symfony 3.3 has similar naming

![default](https://user-images.githubusercontent.com/191082/30025210-041b2d4a-9180-11e7-9dac-a906063f2aa9.png)

Reply:
hm, actually 2.7 has this change too https://github.com/symfony/symfony/pull/20147

Reply:
hmm, indeed. But then, it means you have to add *more* paths to account for them in other places.

Reply:
Done in #1014